inmanta.spec

```diff
@@ -14,67 +18,185 @@
 
 %define sourceversion %{version}%{?buildid}
 %define sourceversion_egg %{version}%{?buildid_egg}
+%define inmanta_core_dir inmanta-core-%{inmanta_core_version}
 
-Name:           python3-inmanta
+Name:           inmanta-oss
 Version:        %{version}
 
 Release:        %{release}%{?buildid}%{?tag}%{?dist}
-Summary:        Inmanta Service Orchestrator
+Summary:        Inmanta automation and orchestration tool
 
 Group:          Development/Languages
-License:        EULA
+License:        ASL 2
 URL:            http://inmanta.com
 Source0:        inmanta-%{sourceversion_egg}.tar.gz
+Source1:        dependencies.tar.gz
+Source2:        inmanta-inmanta-dashboard-%{dashboard_version}.tgz
+Source3:        inmanta-core-%{inmanta_core_version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:  python3-inmanta-core
 BuildRequires:  systemd
+BuildRequires:  sed
+BuildRequires:  libffi-devel
+
+Requires:       git
+Requires:       gcc
+Requires:       logrotate
+Requires:       libffi
+Requires(pre):  shadow-utils
+
+%if 0%{?el7}
+BuildRequires:  openssl11-devel
+Requires:       openssl11
+%else
+BuildRequires:  openssl-devel >= 1:1.1.1
+Requires:       openssl >= 1:1.1.1
+%endif
+
+BuildRequires:  python%{undotted_python_version}-devel
+Requires:       python%{undotted_python_version}
+Requires:       python%{undotted_python_version}-devel
+
+Obsoletes: python3-inmanta
+Obsoletes: python3-inmanta-core
+
+%define __python3 /usr/bin/python%{python_version}
+
+%package -n inmanta-oss-server
+Summary:        The configuration and service files to start the Inmanta server
+Requires:       inmanta-oss
+Obsoletes:      python3-inmanta-server
+
+%package -n inmanta-oss-agent
+Summary:        The configuration and service files to start the Inmanta agent
+Requires:       inmanta-oss
+Obsoletes:      python3-inmanta-agent
 
-Requires:  python3-inmanta-core
+%description
 
-# Use the correct python for bycompiling
-%define __python %{_p3}
+%description -n inmanta-oss-server
 
-%description
+%description -n inmanta-oss-agent
 
 %prep
 %setup -q -n inmanta-%{sourceversion_egg}
+%setup -T -D -a 1 -n inmanta-%{sourceversion_egg}
```

Comment:
why are these three lines?

Reply:
They unpack `Source1`, `Source2` and `Source3` respectively

Reply:
right, -n is the target dir.

Reply:
Yes